scripts/gen_certs.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
```

Comment:
This got the scripts working for me on NixOS but I'm happy to drop this commit if folks don't want to cater to more bizarre setups.